.github/workflows/ci-unix-shared-installed.yml

```diff
@@ -97,3 +97,19 @@ jobs:
               endif()" >> CMakeLists.txt
 
         cmake . -DCMAKE_PREFIX_PATH=../install
+    - name: Prepare libavif with [[nodiscard]] (cmake)
```

Comment:
(no action required) just wondering if this should be in a separate workflow as intuitively it's a different kind of check, but I guess it's faster/more resource efficient to have it here?

Reply:
Indeed, the idea was to minimize resources. Still, that requires to upgrade gcc and forces -Werror (which might be a problem at some point if we have some warnings we do not care about).